Library/EntityDB/Context.cs

```diff
@@ -10,13 +10,13 @@ public class Context : DbContext
         public Context(DbContextOptions<Context> options) : base(options) { }
 
 
-        public virtual DbSet<ChatEntity> DALChat { get; set; }
+        public virtual DbSet<ChatEntity> Chat { get; set; }
```

Comment:
Let's use plural form since it is a collection 
```suggestion
        public virtual DbSet<ChatEntity> Chats { get; set; }
```